### PR TITLE
Implemented infantry units and barracks.

### LINKED ISSUE #155: Implement infantry
Implementing infantry needs a lot of work to make them look/behave properly. This issue tracks all the necessary work.

- [x] ED infantry:
  - [x] Android A02 (rocket launcher), Android A03 (ion cannon)
  - [x] Android A01 (semi-automatic gun, grenades)
  - [x] Android A04 (mines)
- [x] UCS infantry:
  - [x] Silver One (machine gun), Silver R (rocket launcher), Silver Max (plasma cannon)
  - [x] Silver T (kamikaze)

Dependencies:
- [x] #128 (for proper alignment of flipped sprites)
- [x] #6 

Remaining stuff to implement/fix:
- [x] #215 
- [x] #212 
- [x] #216

### PR DESCRIPTION
**Pull request by Dzierzan**
- OpenRA doesn't support random death animations for specific damage type. Issue #182.
- I made duplicate death for UCS and Civils, so they do not disappear when they are killed by unknown damage type (although it should fall back to `die`, but it doesn't).
- ~~Wrong offsets for infantry aiming and shooting, depends on #128.~~
- Used `TurnOnIdle` trait for infantry. While they do not do that in vanilla, I strongly believe it's a nice addon which makes infantry more alive when idle.
- I didn't try to mimic vanilla idle animation behavior, instead I set short sequence which is triggered from time to time using `WithIdleAnimation`.
- Aiming and shooting sequences were seperated in virtual assets.
- Used IceReaper's `WithAimAttackAnimation` for aiming. Technically ORA's `WithAimAnimation` should work too, but it works poorly with `WithAttackAnimation`.
- `WithAimAttackAnimation` needs to be conditonal or support armaments (primary, secondary...) for A01 grenade throw.
- Infantry bodies do not decay. Issue #183
- Implemented custom voices for UCS infantry. Added basically +5% pitch and some reverb effect. I didn't touch "OK" and "Waiting" voices as they are fine.
- Barracks do not have sprites for door animations when they're damaged. So I disabled that animation when a building is damaged. In vanilla, full HP door animation is used for damaged building, it looks just awful.
- ~~`WithProductionDoorOverlay` trait was used for door animation. There's one issue with that. By default it always renders the first frame, the problem is, Earth 2140 doesn't have cut idle sprites for door animations. I patched it by making an extra sprite for both barracks.
  But IMO this solution is not clean and custom trait has to be written. I'd just use `WithProductionDoorOverlay` as base, but do not render the first sprite all the time. Also most likely ZOffset is set quite high for this trait which I cannot overcome with setting own ZOffset. There's apparently value 512
in the trait, if removed, then an infantry actor spawns correctly above building sprite.~~
EDIT: Used this trait instead https://github.com/OpenHV/OpenHV/blob/master/OpenRA.Mods.HV/Traits/Render/WithProductionExitOverlay.cs

#155

### CODE REVIEW

File: OpenRA.Mods.OpenE2140/Traits/Misc/WithAimAttackAnimation.cs [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+using OpenRA.Traits;
+
+[UsedImplicitly(ImplicitUseTargetFlags.WithMembers)]
+public class WithAimAttackAnimationInfo : TraitInfo, Requires<WithSpriteBodyInfo>
+{
+	[Desc("Displayed while attacking.")]
+	[SequenceReference]
+	public readonly string SequenceFire = "";
+
+	[Desc("Displayed while attacking.")]
```
Comment (IceReaper) [inline OpenRA.Mods.OpenE2140/Traits/Misc/WithAimAttackAnimation.cs]: copy & paste?

File: mods/e2140/content/ed/voices.yaml [hunk 2]
```diff
@@ -4,5 +4,14 @@ EdVehicleVoice:
 		Select: 68.smp, 69.smp
 		Action: 70.smp, 71.smp
 		Attack: 74.smp, 75.smp
-		Escort: 73.smp
+		Escort: 72.smp, 73.smp
```
Comment (IceReaper) [inline mods/e2140/content/ed/voices.yaml]: can we move escort before attack? so it makes an ascending order? :)

File: mods/e2140/content/ed/buildings/bio_center/rules.yaml [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+		Facing: 500
+	Exit@Exit3:
+		SpawnOffset: -1000, 500, 0
+		ExitCell: 1,2
+		Facing: 650
+	GrantConditionOnDamageState:
+		Condition: damaged
+	WithProductionDoorOverlay:
+		RequiresCondition: !Transforming && !damaged
+		Sequence: opening
```
Comment (IceReaper) [inline mods/e2140/content/ed/buildings/bio_center/rules.yaml:59]: Encyclopedia :)

File: mods/e2140/content/ucs/buildings/robot_factory/rules.yaml [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+		Facing: 500
+	Exit@Exit3:
+		SpawnOffset: -450, 400, 0
+		ExitCell: 1,2
+		Facing: 650
+	GrantConditionOnDamageState:
+		Condition: damaged
+	WithProductionDoorOverlay:
+		RequiresCondition: !Transforming && !damaged
+		Sequence: opening
```
Comment (IceReaper) [inline mods/e2140/content/ucs/buildings/robot_factory/rules.yaml:59]: Encyclopedia :)

File: mods/e2140/content/ucs/infantry/silver_one/rules.yaml [hunk 5]
```diff
@@ -0,0 +1,25 @@
...
+	Health:
+		HP: 28
+	RevealsShroud:
+		Range: 2c896
+	Armament@PRIMARY:
+		Weapon: ucs_infantry_silver_one
+		MuzzlePalette:
+	EnhancedEncyclopedia:
+		Category: UCS - Infantry
+		Order: 3
```
Comment (IceReaper) [inline mods/e2140/content/ucs/infantry/silver_one/rules.yaml]: Order: 1

File: mods/e2140/content/ucs/infantry/silver_r/rules.yaml [hunk 6]
```diff
@@ -0,0 +1,41 @@
...
+	AmmoPool:
+		Ammo: 2
+		AmmoCondition: ammo
+	ReloadAmmoPool:
+		Delay: 62
+		ResetOnFire: True
+		Count: 1
+	EnhancedEncyclopedia:
+		Category: UCS - Infantry
+		Order: 3
```
Comment (IceReaper) [inline mods/e2140/content/ucs/infantry/silver_r/rules.yaml]: Order: 2

File: mods/e2140/content/ucs/infantry/silver_t/rules.yaml [hunk 7]
```diff
@@ -0,0 +1,34 @@
...
+	Armament@PRIMARY:
+		Weapon: ucs_infantry_silver_t
+		MuzzlePalette:
+	GrantConditionOnAttack:
+		Condition: triggered
+	KillsSelf:
+		RequiresCondition: triggered
+	EnhancedEncyclopedia:
+		Category: UCS - Infantry
+		Order: 3
```
Comment (IceReaper) [inline mods/e2140/content/ucs/infantry/silver_t/rules.yaml]: Order: 4

File: mods/e2140/virtualassets/SPRU0.MIX.VirtualAssets.yaml [hunk 8]
```diff
@@ -442,74 +443,122 @@ Generate:
 		idle: 691-720[5_1_0] 8
 		idle2: 691-720[5_1_0] 8
 		move: 691-720[0_4_2] 8
-		attack: 721-730 8
+			Offsets: 698-700:0,-1 704,706:-1,-1
+		aim:  721-730[0_1_1] 8
```
Comment (IceReaper) [inline mods/e2140/virtualassets/SPRU0.MIX.VirtualAssets.yaml]: single space :-P

File: mods/e2140/virtualassets/SPRU0.MIX.VirtualAssets.yaml [hunk 9]
```diff
@@ -442,74 +443,122 @@ Generate:
...
+			Offsets: 722:8,-1 724:6,0 726:6,0 728:6,0 730:7,0
 
 	infantry_silver_max: Player
 		idle: 731-760[4_1_1] 8
 		idle2: 731-760[5_1_0] 8
+			Offsets: 708:-1,0
 		move: 731-760[0_4_2] 8
-		attack: 761-770 8
+			Offsets: 731,749,755:1,0 738-740,746:-1,0 744:-1,1 756,758:0,-1
+		aim:  761-770[0_1_1] 8
```
Comment (IceReaper) [inline mods/e2140/virtualassets/SPRU0.MIX.VirtualAssets.yaml]: single space :-P

File: mods/e2140/virtualassets/SPRU0.MIX.VirtualAssets.yaml [hunk 10]
```diff
@@ -442,74 +443,122 @@ Generate:
...
+		shoot: 761-770[1_1_0] 8
+			Offsets: 762:6,0 764:5,0 766:6,0 768:6,-1 770:8,0
 
 	infantry_silver_r: Player
 		idle: 771-800[4_1_1] 8
 		idle2: 771-800[5_1_0] 8
 		move: 771-800[0_4_2] 8
-		attack: 801-810 8
+			Offsets: 771,773,774,779,783,791,795,797,798:1,0 778:-1,0
+		aim:  801-810[0_1_1] 8
```
Comment (IceReaper) [inline mods/e2140/virtualassets/SPRU0.MIX.VirtualAssets.yaml]: single space :-P

File: mods/e2140/virtualassets/SPRU0.MIX.VirtualAssets.yaml [hunk 11]
```diff
@@ -442,74 +443,122 @@ Generate:
...
-	infantry_android_a03: Player
+	infantry_android_a03: Player Shadow
 		idle: 851-880[4_1_1] 8
+			Offsets: 851-880:1,0
 		idle2: 851-880[5_1_0] 8
+			Offsets: 851-880:1,0
 		move: 851-880[0_4_2] 8
-		attack: 881-890 8
+			Offsets: 851-880:1,0
+		aim:  881-890[0_1_1] 8
```
Comment (IceReaper) [inline mods/e2140/virtualassets/SPRU0.MIX.VirtualAssets.yaml]: single space :-P

File: mods/e2140/virtualassets/SPRU0.MIX.VirtualAssets.yaml [hunk 12]
```diff
@@ -442,74 +443,122 @@ Generate:
...
 
 	infantry_android_a01: Player
 		idle: 891-920[4_1_1] 8
 		idle2: 891-920[5_1_0] 8
 		move: 891-920[0_4_2] 8
-		attack: 921-930 8
-		attack2: 931-940 8
-
-	infantry_android_a02: Player
+		aim:  921-930[0_1_1] 8
```
Comment (IceReaper) [inline mods/e2140/virtualassets/SPRU0.MIX.VirtualAssets.yaml]: single space :-P

File: mods/e2140/virtualassets/SPRU0.MIX.VirtualAssets.yaml [hunk 13]
```diff
@@ -442,74 +443,122 @@ Generate:
...
+			Offsets: 931:8,0 933:6,0 935,937,939:5,0
+		shoot2: 931-940[1_1_0] 8
+			Offsets: 932:8,0 934:6,0 936,938,940:5,0
+
+	infantry_android_a02: Player 
 		idle: 941-970[4_1_1] 8
 		idle2: 941-970[5_1_0] 8
 		move: 941-970[0_4_2] 8
-		attack: 971-980 8
+		aim:  971-980[0_1_1] 8
```
Comment (IceReaper) [inline mods/e2140/virtualassets/SPRU0.MIX.VirtualAssets.yaml]: single space :-P

**Review by IceReaper — COMMENTED** [review]
Why do we have custom sounds added? Wont that generate problems with different translations?

File: OpenRA.Mods.OpenE2140/Traits/Misc/WithAimAttackAnimation.cs (hunk 1 above)
Comment (Dzierzan) [reply OpenRA.Mods.OpenE2140/Traits/Misc/WithAimAttackAnimation.cs]: Yes, you know I'm no C# coder.

File: mods/e2140/content/ed/voices.yaml (hunk 2 above)
Comment (Dzierzan) [reply mods/e2140/content/ed/voices.yaml]: Maybe for another PR?

File: mods/e2140/content/ed/buildings/bio_center/rules.yaml (hunk 3 above)
Comment (Dzierzan) [reply mods/e2140/content/ed/buildings/bio_center/rules.yaml:53]: No idea what you mean.

File: mods/e2140/content/ucs/buildings/robot_factory/rules.yaml (hunk 4 above)
Comment (Dzierzan) [reply mods/e2140/content/ucs/buildings/robot_factory/rules.yaml:53]: No idea what you mean.

**Comment by Dzierzan** [comment]
You expect supporting translations? Besides if so, it will just fall back to the translated voices. I only added minor improvements, but I didn't create totally new voice (like I did in OpenKrush).

File: mods/e2140/content/ed/infantry/a02/rules.yaml [hunk 14]
```diff
@@ -0,0 +1,41 @@
...
+		ResetOnFire: True
+		Count: 1
+	EnhancedEncyclopedia:
+		Category: ED - Infantry
+		Order: 2
+		Animation: DATABASE/A02.FLC
+		Title: A02 Android
+		Armor: None
+		Armament: Rocket launcher
+		Description: The A02 is an enhanced version of the A01. They have the same features and differ only in weaponry. The A02 has a rocket launcher and is ideal for destroying heavyweight enemy vehicles. You can also use it for ground to air combat, its wide range rocket launcher enabling you to hit air-borne targets.
```
Comment (michaeldgg2) [inline mods/e2140/content/ed/infantry/a02/rules.yaml]: Nitpick:

```suggestion
		Description: The A02 is an enhanced version of the A01. They have the same features and differ only in weaponry. The A02 has a rocket launcher and is ideal for destroying heavyweight enemy vehicles. You can also use it for ground to air combat, its wide range rocket launcher enabling you to hit airborne targets.
```

File: mods/e2140/content/ed/voices.yaml (hunk 2 above)
Comment (michaeldgg2) [reply mods/e2140/content/ed/voices.yaml]: Guard voice line (`72.smp`) is played in Earth 2140 when *Guard* command is issued. This is different than Guard in OpenRA. This Guard command (in Earth 2140) is somewhat similar to a mix between Attack Anything and Defend unit stances: unit defends the spot, where the command was activated and chases enemies but only to certain distance. A bit off topic, but still it's good to clarify things.

Generally it's better if PR is as small as possible and this change is not related to infantry.

File: mods/e2140/content/ucs/buildings/robot_factory/rules.yaml [hunk 15]
```diff
@@ -0,0 +1,87 @@
+ucs_buildings_robot_factory:
+	Inherits: ^CoreBuilding
+	Tooltip:
+		Name: Light Robot Factory
```
Comment (michaeldgg2) [inline mods/e2140/content/ucs/buildings/robot_factory/rules.yaml]: It's called only *Robot Factory*. Yes, the robots it produces are definitely lighter than those produced by Heavy Tech, but let's not change stuff just for the sake of it. For me as fan and player of Earth 2140 it suggests it's some sort of new/different building.

```suggestion
		Name: Robot Factory
```

File: mods/e2140/content/ucs/buildings/robot_factory/rules.yaml [hunk 16]
```diff
@@ -0,0 +1,87 @@
...
+		Sequence: opening
+		RequiresCondition: !Transforming && !damaged
+	RallyPoint:
+		Cursor: move
+		LineWidth: 2
+	EnhancedEncyclopedia:
+		Category: UCS - Buildings
+		Order: 2
+		Animation: DATABASE/X18.FLC
+		Title: UCS Light Robot Factory
```
Comment (michaeldgg2) [inline mods/e2140/content/ucs/buildings/robot_factory/rules.yaml]: Same as above, it's just *Robot Factory*.

```suggestion
		Title: UCS Robot Factory
```

File: mods/e2140/content/ucs/buildings/robot_factory/rules.yaml [hunk 17]
```diff
@@ -0,0 +1,87 @@
...
+		Armor: Protected from traditional weapons
+		Armament: None
+		Resistance: Medium
+		Description: Small battle units of the SILVER type are produced in this building. The whole production process happens underground. The completed robots are then brought to the surface. This underground construction makes the light-robot factory more attack proof than a building on the surface would be.
+
+ucs_mcu_robot_factory:
+	Inherits@1: ^SharedVehicleMcu
+	Tooltip:
+		Name: Mobile Light Robot Factory
```
Comment (michaeldgg2) [inline mods/e2140/content/ucs/buildings/robot_factory/rules.yaml]:
```suggestion
		Name: Mobile Robot Factory
```

File: mods/e2140/content/core/rules/defaults.yaml [hunk 18]
```diff
@@ -213,29 +213,55 @@
...
 	# Calculated size for infantry to match shared cells.
 	Selectable:
-		Bounds: 576,640,0,-192
+		Bounds: 256,288,0,0
 	# Infantry hitshape
 	HitShape:
 		Type: Circle
 			Radius: 128
-	# Infantry has an animation while moving: walking.
+	# Infatry has an animation while moving.
```
Comment (michaeldgg2) [inline mods/e2140/content/core/rules/defaults.yaml]:
```suggestion
	# Infantry has an animation while moving.
```

File: mods/e2140/content/shared/buildings/air_base/rules.yaml [hunk 19]
```diff
@@ -7,6 +7,7 @@ shared_buildings_air_base:
 	Building:
 		Dimensions: 4,3
 		Footprint: =xxx xxx= ====
+		LocalCenterOffset: 0,0,-256
```
Comment (michaeldgg2) [inline mods/e2140/content/shared/buildings/air_base/rules.yaml]: Why is this necessary? Does original game have the building's sprite shifted like this?

File: mods/e2140/content/ucs/buildings/robot_factory/rules.yaml [hunk 20]
```diff
@@ -0,0 +1,87 @@
...
+			TopLeft: -950, -700
+			BottomRight: 900, 750
+	Selectable:
+		Bounds: 2064, 1984, 0,176
+	Power:
+		Amount: -200
+	Production:
+		Produces: Infantry.UCS
+		RequiresCondition: !Transforming
+	ProductionQueue:
```
Comment (michaeldgg2) [inline mods/e2140/content/ucs/buildings/robot_factory/rules.yaml:30]: Use `^CoreFactory` that defines this basic stuff for production buildings. It also adds `RallyPoint` trait.

Yes there's separate `^CoreElevatorFactory` which is used for factories with elevators only of course.

File: mods/e2140/content/ed/buildings/bio_center/rules.yaml [hunk 21]
```diff
@@ -0,0 +1,87 @@
+ed_buildings_bio_center:
+	Inherits: ^CoreBuilding
+	Tooltip:
+		Name: Bionic Production Center
```
Comment (michaeldgg2) [inline mods/e2140/content/ed/buildings/bio_center/rules.yaml]: Similar to Robot Factory, please use original name: *Bio-Center* (with or without hyphen).

```suggestion
		Name: Bio-Center
```

File: mods/e2140/content/ed/buildings/bio_center/rules.yaml [hunk 22]
```diff
@@ -0,0 +1,87 @@
...
+			TopLeft: -1000, -900
+			BottomRight: 900, 750
+	Selectable:
+		Bounds: 2064, 1984, -32,-32
+	Power:
+		Amount: -200
+	Production:
+		Produces: Infantry.ED
+		RequiresCondition: !Transforming
+	ProductionQueue:
```
Comment (michaeldgg2) [inline mods/e2140/content/ed/buildings/bio_center/rules.yaml:30]: Use `^CoreFactory` similar to Robot Factory

File: mods/e2140/content/ed/buildings/bio_center/rules.yaml [hunk 23]
```diff
@@ -0,0 +1,87 @@
...
+		Sequence: opening
+		RequiresCondition: !Transforming && !damaged
+	RallyPoint:
+		Cursor: move
+		LineWidth: 2
+	EnhancedEncyclopedia:
+		Category: ED - Buildings
+		Order: 2
+		Animation: DATABASE/X10.FLC
+		Title: Bionic Production Center (Bio Center)
```
Comment (michaeldgg2) [inline mods/e2140/content/ed/buildings/bio_center/rules.yaml]:
```suggestion
		Title: Bio-Center
```

File: mods/e2140/content/ed/infantry/a01/weapons.yaml [hunk 24]
```diff
@@ -0,0 +1,77 @@
...
+		ImpactSounds: 20.smp, 21.smp
+		ValidTargets: Water
+		InvalidTargets: Vehicle, Structure, Infantry
+	Warhead@EffectSmoke: CreateEffect
+		Image: smoke_firearm
+		Explosions: smoke_firearm1, smoke_firearm2, smoke_firearm3
+		ExplosionPalette:
+		ValidTargets: Vehicle, Structure, Infantry
+
+ed_infantry_a01_machine_gun_vs_infantry:
```
Comment (michaeldgg2) [inline mods/e2140/content/ed/infantry/a01/weapons.yaml]: Why is this separate weapon necessary? Android A01 shoots infantry with machine gun. Also whether machine gun or grenades are used when shooting at ground, it actually depends on what was previous target: if it was infantry, then machine gun is used, if vehicle/structure/etc. then grenades (not sure about trees and other terrain objects though).

File: mods/e2140/content/ed/infantry/a01/rules.yaml [hunk 25]
```diff
@@ -0,0 +1,36 @@
+ed_infantry_a01:
+	Inherits@1: ^EdInfantry
+	Inherits@2: ^CoreArmedSelfInfantry
+	Tooltip:
+		Name: A01 Android
```
Comment (michaeldgg2) [inline mods/e2140/content/ed/infantry/a01/rules.yaml]: Use original name *Android A01*:

```suggestion
		Name: Android A01
```

File: mods/e2140/content/ed/infantry/a02/rules.yaml [hunk 26]
```diff
@@ -0,0 +1,41 @@
+ed_infantry_a02:
+	Inherits@1: ^EdInfantry
+	Inherits@2: ^CoreArmedSelfInfantry
+	Tooltip:
+		Name: A02 Android
```
Comment (michaeldgg2) [inline mods/e2140/content/ed/infantry/a02/rules.yaml]: Use original name *Android A02*:

```suggestion
		Name: Android A02
```

File: mods/e2140/content/ed/infantry/a03/rules.yaml [hunk 27]
```diff
@@ -0,0 +1,26 @@
+ed_infantry_a03:
+	Inherits@1: ^EdInfantry
+	Inherits@2: ^CoreArmedSelfInfantry
+	Tooltip:
+		Name: A03 Android
```
Comment (michaeldgg2) [inline mods/e2140/content/ed/infantry/a03/rules.yaml]: Use original name *Android A03*:

```suggestion
		Name: Android A03
```

File: mods/e2140/content/ed/infantry/a04/rules.yaml [hunk 28]
```diff
@@ -0,0 +1,22 @@
+ed_infantry_a04:
+	Inherits: ^EdInfantry
+	Tooltip:
+		Name: A04 Android
```
Comment (michaeldgg2) [inline mods/e2140/content/ed/infantry/a04/rules.yaml]: Use original name *Android A04*:

```suggestion
		Name: Android A04
```

File: mods/e2140/content/ed/infantry/a04/rules.yaml [hunk 29]
```diff
@@ -0,0 +1,22 @@
...
+	Health:
+		HP: 20
+	RevealsShroud:
+		Range: 2c896
+	#TODO Add mines
+	EnhancedEncyclopedia:
+		Category: ED - Infantry
+		Order: 4
+		Animation: DATABASE/A04.FLC
+		Title: A04 Android
```
Comment (michaeldgg2) [inline mods/e2140/content/ed/infantry/a04/rules.yaml]: Use original name *Android A04*:

```suggestion
		Title: Android A04
```

File: mods/e2140/content/ed/infantry/a03/rules.yaml [hunk 30]
```diff
@@ -0,0 +1,26 @@
...
+		Range: 2c896
+	Armament@PRIMARY:
+		Weapon: ed_infantry_a03
+		LocalOffset: 400,0,50
+		MuzzlePalette:
+	EnhancedEncyclopedia:
+		Category: ED - Infantry
+		Order: 3
+		Animation: DATABASE/A03.FLC
+		Title: A03 Android
```
Comment (michaeldgg2) [inline mods/e2140/content/ed/infantry/a03/rules.yaml]: Use original name *Android A03*:

```suggestion
		Title: Android A03
```

File: mods/e2140/content/ed/infantry/a02/rules.yaml [hunk 31]
```diff
@@ -0,0 +1,41 @@
...
+		AmmoCondition: ammo
+	ReloadAmmoPool:
+		Delay: 62
+		ResetOnFire: True
+		Count: 1
+	EnhancedEncyclopedia:
+		Category: ED - Infantry
+		Order: 2
+		Animation: DATABASE/A02.FLC
+		Title: A02 Android
```
Comment (michaeldgg2) [inline mods/e2140/content/ed/infantry/a02/rules.yaml]: Use original name *Android A02*:

```suggestion
		Title: Android A02
```

File: mods/e2140/content/ed/infantry/a01/rules.yaml [hunk 32]
```diff
@@ -0,0 +1,36 @@
...
+		Name: tertiary
+		Weapon: ed_infantry_a01_grenade
+		MuzzlePalette:
+	AttackFrontal:
+		Armaments: primary, secondary, tertiary
+	EnhancedEncyclopedia:
+		Category: ED - Infantry
+		Order: 1
+		Animation: DATABASE/A01.FLC
+		Title: A01 Android
```
Comment (michaeldgg2) [inline mods/e2140/content/ed/infantry/a01/rules.yaml]: Use original name *Android A01*:

```suggestion
		Title: Android A01
```

**Review by michaeldgg2 — CHANGES_REQUESTED** [review]

File: mods/e2140/content/ed/infantry/a01/weapons.yaml (hunk 24 above)
Comment (Dzierzan) [reply mods/e2140/content/ed/infantry/a01/weapons.yaml]: I miscalculated that. It's fixed.

**Pull request closed without merging**